content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,346 @@
+* `data_transfer_bytes` &#8212; Sets the maximum amount of egress data, measured in bytes, that can be transferred out of Neon from across all of a project's branches using the proxy.
+
+There is one additional `quota` parameter, `logical_size_bytes`, which applies to individual branches, not to the overall project. You can use `logical_size_bytes` to set the maximum size (measured in bytes) that any one individual branch is allowed to reach. Once this threshold is met, the compute for that particular branch (and _only_ that particular branch) is suspended. Note that this limit is _not_ refreshed once per month: it is a strict size limit that applies for the life of the branch.
+
+### Sample quotas
+
+Let's say you want to set limits for an application with two tiers, Trial and Pro, you might set limits like the following:
+
+| Parameter (project)                 | Trial                     | Pro                      |
+|------------------------|---------------------------|--------------------------|
+| `active_time_seconds`  | 86,400 (1 day)            | 2,592,000 (30 days)      |
+| `compute_time_seconds` | 3,600 (1 hour)            | 50,000 (approx. 14 hours)|
+| `written_data_bytes`   | 1,000,000,000 (approx. 1 GiB)| 50,000,000,000 (approx. 50 GiB) |
+| `data_transfer_bytes`  | 500,000,000 (approx. 500 MiB)| 10,000,000,000 (approx. 10 GiB)  |
+
+| Parameter (branch)             | Trial                    | Pro                      |
+|------------------------|---------------------------|--------------------------|
+| `logical_size_bytes`   | 100,000,000 (approx. 100 MiB)| 10,000,000,000 (approx. 10 GiB)  |
+
+
+### Guidelines
+
+Generally, the most effective quotas for controlling spend per project are those controlling maximum compute (`active_time_seconds` and `compute_time_seconds`) and maximum written storage (`written_data_bytes`). In practice, it is possible that `data_transfer_bytes` could introduce unintended logical constraints against your usage. For example, let's say you want to run a cleanup operation to reduce your storage. If part of this cleanup operation involves moving data across the network (for instance, to create an offsite backup before deletion), the `data_transfer_bytes` limit could prevent you from completing the operation &#8212; an undesirable situation where two measures meant to control cost interfere with one another.
+
+## Suspending active computes
+
+_**What happens when the quota is met?**_
+
+When any configured metric reaches its quota limit, all active computes for that project are automatically suspended. It is important to understand, this suspension is persistent. It works differently than the inactivity-based [autosuspend](/docs/guides/auto-suspend-guide), where computes restart at the next interaction: this suspend will _not_ restart at the next API call or incoming proxy connection. Without intervention, the suspension remains in place until the next billing period starts (`quota_reset_at`).
```

Comment:
Should we mention who would have to intervene? Is it Neon Support or the partner?

Reply:
It's on the partner. Revised to:

"If you don't take explicit action otherwise, the suspension remains in place until the end of the current billing period starts (`consumption_period_end`)."